build/templates/products/client.yaml

```diff
@@ -8,15 +8,14 @@ HALE:
   LauncherName: HALE
   ProgramArgs: -nl en_GB -clean -name "HUMBOLDT Alignment Editor" --launcher.defaultAction openFile
   VmArgs:
-  - -Xmx800m
```

Comment:
Any specific reasons why these were removed?

Reply:
Don't remember exactly, however, I tested the build creation and launching of the application with these reverted and it worked. So reverted the changes.